padvision/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .padvision import PadVision
+from .padvision import PadVision, NeuralClassifierBoardExtractor
```

Comment:
unrelated?

Reply:
Same issue as the `from .dadguide import *`, but less related to the whole thing.